src/Spryker/Client/RabbitMq/RabbitMqClient.php

```diff
@@ -21,8 +23,20 @@ class RabbitMqClient extends AbstractClient implements RabbitMqClientInterface
      *
      * @return \Spryker\Client\Queue\Model\Adapter\AdapterInterface
      */
-    public function createQueueAdapter()
+    public function createQueueAdapter(): AdapterInterface
```

Comment:
Please, do not change such things in the public api. This can be considered as a BC break

Reply:
reverted